examples/flutter_gallery/lib/demo/pan_and_zoom_demo.dart

```diff
@@ -0,0 +1,131 @@
+import 'dart:ui' show Vertices;
+import 'package:flutter/material.dart';
+import 'pan_and_zoom_demo_board.dart';
+import 'pan_and_zoom_demo_edit_board_point.dart';
+import 'pan_and_zoom_demo_transform_interaction.dart';
+
+class PanAndZoomDemo extends StatefulWidget {
+  const PanAndZoomDemo({Key key}) : super(key: key);
+
+  static const String routeName = '/pan_and_zoom';
+
+  @override _PanAndZoomDemoState createState() => _PanAndZoomDemoState();
+}
+class _PanAndZoomDemoState extends State<PanAndZoomDemo> {
+  static const double _kHexagonRadius = 32.0;
+  static const double _kHexagonMargin = 1.0;
+  static const int _kBorderRadius = 8;
+
+  bool _reset = false;
+  Board _board = Board(
+    boardRadius: _kBorderRadius,
+    hexagonRadius: _kHexagonRadius,
+    hexagonMargin: _kHexagonMargin,
+  );
+
+  @override
+  Widget build (BuildContext context) {
+    final BoardPainter painter = BoardPainter(
+      board: _board,
+    );
+    final Size screenSize = MediaQuery.of(context).size;
```

Comment:
Desktops...are there any plans of officially supporting Flutter on desktops? 😮

No need to answer but I am curious if you want to respond.

Reply:
Sounds good, I wasn't aware of that pattern but I like it a lot more.

I may revisit the `size` and `visibleRect` params when I bring this widget into Flutter.  It might be better if this is as big as it can be and users can constrain the size with a parent widget.